youtbestyle2.css

```diff
@@ -4,7 +4,7 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    /* justify-content: space-between; */
+
```

Comment:
I think it is always better to remove the comments from the code base, this will make the code look clean.

Reply:
[Optional] It's recommended to make your review comments look more friendly by adding some emojis 😊